src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends BaseShape {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration.